### PR TITLE
Sensei cmake config path

### PR DESCRIPTION
**Pull request by kwryankrattiger**
Solves #79 

@ax3l

### CODE REVIEW

File: CMake/config.cmake [hunk 1]
```diff
@@ -4,12 +4,14 @@ install(FILES ${CMAKE_BINARY_DIR}/senseiConfig.h DESTINATION include)
 
 set(SENSEI_BUILD OFF)
 
+set(sensei_CMAKE_INSTALL_CMAKEDIR ${sensei_CMAKE_INSTALL_CMAKEDIR}/sensei)
```
Comment (ax3l) [inline CMake/config.cmake:7]: Is `sensei_CMAKE_INSTALL_CMAKEDIR` set at this point already (to something like `lib/cmake/`)?

Comment (kwryankrattiger) [reply CMake/config.cmake:7]: Copy-paste error!

**Comment by kwryankrattiger** [comment]
@burlen I am looking into why the python wheel is broken but I think this change is otherwise okay to merge

**Review by burlen — COMMENTED** [review]
looks good!

**Pull request merged**